docs/filters.rst

```diff
@@ -157,6 +157,41 @@ filters:
  * method filter that is related to a method
  * field filter that is related to a model field
 
+.. class:: pyston.filters.default_filters.Filter
+
+  .. attribute:: identifiers
+
+    List of filter terms of the filter.
+
+  .. attribute:: identifiers_suffix
+
+    Suffix usually contains the operator that will be used for filtering. But it can contains more values. For example ``identifiers_suffix`` for ``DateFilter`` can be ``['day', 'gt']``.
+
+  .. attribute:: identifiers_prefix
+
+    List of filter terms over which the filter was found. Because filters can be used recursive (User can be filtered via its Issues). For example if there is foreign key between ``Issue`` and ``User`` named ``created_by``, issue can be filtered with term ``created_by__name__gt``. Values of ``identifiers_prefix``, ``identifiers``, ``identifiers_suffix`` will be:
+
+      * ``identifiers_prefix = ['created_at']``
+      * ``identifiers = ['name']``
+      * ``identifiers_suffix = ['gt']``
+
+  .. attribute:: full_identifiers
+
+    Attribute is combination of ``identifiers_prefix + identifiers + identifiers_suffix``.
+
+  .. attribute:: model
+
+    Django model class according to is filtered.
+
+  .. attribute:: field
+
+    Attribute ``field`` will be set only if filter is instance of ``FieldFilter`` and contains model field that is filtered.
+
+  .. attribute:: method
+
+    Like to ``field`` the attribute is set only if filter is instance of ``MethodFilter`` and contains model method that is filtered.
```

Comment:
`field` a `method` jsou jen read only properties?

Reply:
všechny vstupy jsou readonly

Reply:
Myslím to tak že si je nemůžeš nadefinovat, ne že jdou měnit v runtime. Z toho popisu to chápu tak, že je nastavuje sám filtr.

Reply:
ano nastavuje je ten kdo filtr vytváří, tedy filtr manager, ty je definovat nemůžeš a to ani staticky